yin_yang/plugins/kvantum.py

```diff
@@ -10,6 +12,20 @@ def __init__(self):
         self.theme_light = 'KvFlatLight'
         self.theme_dark = 'KvFlat'
 
+    def set_theme(self, theme: str):
```

Comment:
This is nice, but the only benefit I can see of using dbus instead of the command line is that this works in flatpaks. This would be great, but we would need to convert the whole plugin from a command line based one to a standard plugin in order to get everything working inside a flatpak. This requires a bit more work, so maybe seperate this into a new pull request.

Reply:
OK, I will revert it and make a new pr when I an ready